extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -85,8 +86,10 @@ class AmpInstagram extends AMP.BaseElement {
     const image = this.win.document.createElement('amp-img');
     // This will redirect to the image URL. By experimentation this is
     // always the same URL that is actually used inside of the embed.
-    image.setAttribute('src', 'https://www.instagram.com/p/' +
-        encodeURIComponent(this.shortcode_) + '/media/?size=l');
+    const imgSrc = resolveUrlAttr('amp-img', 'src',
```

Comment:
so i think last time i mentioned the only way I know of how to get the cdn url is to hit an endpoint:

`http://api.instagram.com/oembed?callback=&url=https%3A%2F%2Finstagram.com%2Fp%2FfBwFP`

Would that be viable?